osu.Game.Rulesets.Tau/Objects/Drawables/DrawableSliderTick.cs

```diff
@@ -0,0 +1,58 @@
+﻿using osu.Game.Rulesets.Scoring;
+using osu.Framework.Graphics;
+
+#if SHOW_TICKS
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osuTK;
+using osuTK.Graphics;
+#endif
+
+namespace osu.Game.Rulesets.Tau.Objects.Drawables
+{
+    public class DrawableSliderTick : DrawableBeat
+    {
+        public DrawableSlider DrawableSlider => (DrawableSlider)ParentHitObject;
+
+        public override bool DisplayResult => false;
+
+        public DrawableSliderTick()
+        {
+        }
+
+        public DrawableSliderTick(Beat hitObject)
+            : base(hitObject)
+        {
+        }
+
+        protected override Drawable CreateDrawable()
+        {
+#if SHOW_TICKS
+            return new Container
+            {
+                RelativePositionAxes = Axes.Both,
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                Alpha = 0,
+                AlwaysPresent = true,
+                Size = new Vector2(NoteSize.Default),
+                Child = new Circle
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Colour = Color4.Red
+                }
+            };
+#else
```

Comment:
This can be toggled on if you add `#define SHOW_TICKS` at the very top of the file. I've left this here as we may need to go back and see if the ticks are in the correct position if we ever do mess with the slider generator or something similar.